### PR TITLE
readme for azure core samples

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/core/azure-core/samples/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Azure Core Samples
+
+test_example_sync.py - samples of how to create a sync pipeline
+
+test_example_async.py - samples of how to create an async pipeline
+
+test_example_sansio.py - samples of how to config policies
```
Comment (bryevdv) [inline sdk/core/azure-core/samples/README.md]: would suggest applying formatting to filenames (emph or pre) will make things a little easier to distinguish visually

File: sdk/core/azure-core/samples/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+topic: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-core
+---
+
+# Azure Core Samples
```
Comment (kaerm) [inline sdk/core/azure-core/samples/README.md]: Azure Core Library Python Samples

File: sdk/core/azure-core/samples/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+topic: sample
```
Comment (kaerm) [inline sdk/core/azure-core/samples/README.md]: page_type instead of topic

File: sdk/core/azure-core/samples/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+languages:
+  - python
+products:
+  - azure
+  - azure-core
```
Comment (kaerm) [inline sdk/core/azure-core/samples/README.md]: core is not a product, delete the line and keep only azure

**Pull request merged**